pkg/controller/networkpolicy/validate.go

```diff
@@ -760,6 +849,22 @@ func validateAntreaGroupSpec(s crdv1alpha2.GroupSpec) (string, bool) {
 	if selector+serviceRef+ipBlock+ipBlocks+childGroups > 1 {
 		return errMsg, false
 	}
+	multicast := false
+	unicast := false
+	for _, ipb := range s.IPBlocks {
+		ipaddr, _, err :=net.ParseCIDR(ipb.CIDR)
+		if err != nil {
+			continue
+		}
+		if ipaddr.IsMulticast() {
+			multicast = true
+		} else {
+			unicast = true
+		}
+		if multicast && unicast {
+			return "can not set multicast ipaddress together with unicast ip address", false
```

Comment:
Since we use different pipeline for multicast and unicast, set multicast and unicast address at the same time in IPBlocks will make the logic more complicated.

Reply:
done